src/utils/tree.js

```diff
@@ -41,9 +42,16 @@ var tf =
         } else {
           //found a leave
           let seq = seqs.filter(s => s.name === nwck.name)[0];
+
           if(seq != null){
-            seq.ids = [`s${seq.id + 1}`];
-            nwck.name = `s${seq.id + 1}`;
+            if(typeof seq.id === 'number'){
```

Comment:
yes but my point was that it could be that at some point the tree also has numerical ids.